app/views/pseuds/delete_preview.html.erb

```diff
@@ -1,28 +1,27 @@
-<h2 class="heading"><%=h 'Pseud deletion' %></h2>
+<h2 class="heading"><%=h ts("Pseud deletion") %></h2>
 
 <p>
-  When you delete your pseud, any works, series, or comments you have created under it will be 
-  transferred to your default pseud.
+  <%= ts("When you delete your pseud, any works, series, or comments you have created under it will be 
```

Comment:
We're moving away from using `ts`, which is [a custom setup we never quite finished](https://github.com/otwcode/otwarchive/blob/14fbeb6ae13f9637cfdd3c4f586620d8863ff32a/config/initializers/monkeypatches/translate_string.rb#L121), towards doing i18n [how Rails would expect](https://guides.rubyonrails.org/i18n.html). That means we use the `t` helper, and set up the English strings in config/locales/views/en.yml.

You can see how we did that for user deletion preview:

- https://github.com/otwcode/otwarchive/blob/14fbeb6ae13f9637cfdd3c4f586620d8863ff32a/app/views/users/delete_preview.html.erb
- https://github.com/otwcode/otwarchive/blob/14fbeb6ae13f9637cfdd3c4f586620d8863ff32a/config/locales/views/en.yml#L142-L162

Our i18n house style is documented at https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards.